testing/btest/Files/redis.conf

```diff
@@ -0,0 +1,2295 @@
+# Redis configuration file example.
```

Comment:
Would it make sense to kick out (almost) anything commented in this file? It's just a test case, so I think most of the helpful context for a Redis admin isn't really required.